broker-daemon/bin/sparkswapd.js

```diff
@@ -5,97 +5,104 @@
-  .option('--rpc-user [rpc-user]', 'Rpc user name, used when auth is enabled', program.String, RPC_USER)
-  .option('--rpc-pass [rpc-pass]', 'Rpc password, used when auth is enabled', program.String, RPC_PASS)
-  .option('--rpc-privkey-path [path>', 'Location of private key for the broker\'s rpc', validations.isFormattedPath, RPC_PRIV_KEY)
-  .option('--rpc-pubkey-path [path]', 'Location of the public key for the broker\'s rpc', validations.isFormattedPath, RPC_PUB_KEY)
-  .option('--disable-relayer-auth [disable-relayer-auth]', 'Disable SSL and message signing to the relayer (DEV ONLY)', program.BOOL, DISABLE_RELAYER_AUTH)
-  .option('--id-privkey-path [path]', 'Location of private key for the broker\'s identity', validations.isFormattedPath, ID_PRIV_KEY)
-  .option('--id-pubkey-path [path]', 'Location of the public key for the broker\'s identity', validations.isFormattedPath, ID_PUB_KEY)
-  .option('--markets [markets]', 'Comma-separated market names to track on startup', validations.areValidMarketNames, MARKETS)
-  .option('--relayer-host [server]', 'The host address for the SparkSwap Relayer', validations.isHost, RELAYER_RPC_HOST)
-  .option('--relayer-cert-path [path]', 'Location of the root certificate for the SparkSwap Relayer', validations.isFormattedPath, RELAYER_CERT_PATH)
+  // Broker configuration options
+  .option('--broker.data-dir [data-dir]', 'Location to store SparkSwap data', validations.isFormattedPath, config.dataDir)
+  .option('--broker.interchain-router-address [interchain-router-address]', 'Add a host/port to listen for interchain router RPC connections', validations.isHost, config.interchainRouterAddress)
+  .option('--broker.id-pub-key [id-pub-key]', 'Location of the public key for the broker\'s identity', validations.isFormattedPath, config.idPubKey)
+  .option('--broker.id-priv-key [id-priv-key]', 'Location of private key for the broker\'s identity', validations.isFormattedPath, config.idPrivKey)
+  // TODO: Make this a list instead of string
+  .option('--broker.markets [markets]', 'Comma-separated market names to track on startup', validations.areValidMarketNames, config.markets)
+  .option('--broker.disable-auth [disable-auth]', 'Disable SSL for the broker (DEV ONLY)', program.BOOL, config.disableAuth)
+  // Relayer configuration options
+  .option('--relayer.rpc-host [rpc-host]', 'The host address for the SparkSwap Relayer', validations.isHost, config.relayer.rpcHost)
+  .option('--relayer.cert-path [cert-path]', 'Location of the root certificate for the SparkSwap Relayer (only used in development)', validations.isFormattedPath, config.relayer.certPath)
+  // Broker RPC configuration options
+  .option('--rpc.address [rpc-address]', 'Add a host/port to listen for daemon RPC connections', validations.isHost, config.rpc.rpcAddress)
+  .option('--rpc.rpc-user [rpc-user]', 'Broker rpc user name', program.String, config.rpc.rpcUser)
```

Comment:
this option is still `--rpc.rpc-user` which is duplicative and inconsistent with `--rpc.address` above